Stripe/PublicHeaders/STPKlarnaLineItem.h

```diff
@@ -0,0 +1,73 @@
+//
+//  STPKlarnaLineItem.h
+//  Stripe
+//
+//  Created by David Estes on 11/19/19.
+//  Copyright © 2019 Stripe, Inc. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ The type of the Klarna line item.
+*/
+typedef NS_ENUM(NSUInteger, STPKlarnaLineItemType) {
+    /**
+     The line item for a product
+     */
+    STPKlarnaLineItemTypeSku,
```

Comment:
```suggestion
    STPKlarnaLineItemTypeSKU,
```

Reply:
I had it this way originally and changed it, but I forgot why. I'll try it again!